tools/run_tests/run_interop_tests.py

```diff
@@ -1515,6 +1506,8 @@ def aggregate_http2_results(stdout):
                                 transport_security = 'compute_engine_channel_creds'
                             else:
                                 transport_security = 'tls'
+                            if transport_security not in args.custom_credentials_type:
```

Comment:
does this `continue` need to be here? I can't see when it would ever get hit

Reply:
if I restrict the custom credential types used by a cmdline argument, it might skip the e.g. the compute_engine_channel_creds test, which I think is desirable. Am I missing something?